src/Pages/FarmTImeLine.js

```diff
@@ -79,118 +106,144 @@ function FarmTimeline() {
     );
   };
 
+  let timeLine = processHistory.sortBy(function (o) { return [ o.startDueDate ] }).map(function (obj, index) {
+
+    return (<TimeLine key={index} process={<Card
+      processName={obj.processName.displayStr}
+      processStatus={obj.processStatus.displayStr}
+      startDueDate={obj.startDueDate.substring(0, 4)+" - "+months[parseInt(obj.startDueDate.substring(obj.startDueDate.indexOf("-")+1 ,obj.startDueDate.indexOf("-")+3 ), 10)]+" - "+obj.startDueDate.substring(obj.startDueDate.lastIndexOf("-")+1 ,obj.startDueDate.lastIndexOf("-")+3 )}
+      endDueDate={obj.endDueDate.substring(0, 4)+" - "+ months[parseInt(obj.endDueDate.substring(obj.endDueDate.indexOf("-")+1 ,obj.endDueDate.indexOf("-")+3 ),10)]+" - "+obj.endDueDate.substring(obj.endDueDate.lastIndexOf("-")+1 ,obj.endDueDate.lastIndexOf("-")+3 )}
+    
+    />}>
+    </TimeLine>
+    );
```

Comment:
Please reformat the code, probably add some documentation, to make it easier to read and understand.
You can create a custom Card component which takes `process` object as prop and does all these things internally.